### DISCUSSION TITLE
SuperCooKey – A SuperCookie Built Into TLS 1.2 and 1.3

### DISCUSSION THREAD

**Discussion by nchv** [Eloston/ungoogled-chromium#1902] [Ideas]
I am not aware if this has been addressed in upstream Chromium but I thought it may be worth mentioning (perhaps another flag to implement?):

https://www.privateinternetaccess.com/blog/2018/11/supercookey-a-supercookie-built-into-tls-1-2-and-1-3/

**Comment by xsmile** [comment]
AFAIK 0-RTT is not implemented in Chromium right now and this should not be an issue. This might change with future releases though.

**Comment by nchv** [comment]
You may be right but considering how serious this sounds it deserves a more in-depth look.

**Comment by bsd-source** [comment]
Wow nice find anchev. Looks like they are actively working on it right now in chromium.

[Interesting flags](https://github.com/chromium/chromium/blob/0c489f4f41545adab3d4682f0f3c99713a3cb062/net/quic/quic_flags_list.h)

**Comment by nchv** [comment]
@azdps the link you shared seems to be about QUIC flags. How is that related?

BTW speaking of QUIC: I see it as the next anti-privacy thing by G because Tor works with TCP only. Of course that's a whole other big topic to discuss separately.

**Comment by Atavic** [comment]
Zero Round Trip Time Resumption (0-RTT) is included into QUIC, an Alphabet (Google) effort.
A good explanation is [here](https://blog.erratasec.com/2018/11/some-notes-about-http3.html)
